components/sup/src/event/error.rs

```diff
@@ -1,9 +1,10 @@
 //! Event subsystem-specific error handling
 
-use rants::{error::Error as RantsError,
-            native_tls};
+use native_tls;
+use tokio::time::error::Elapsed;
 use std::{error,
           fmt,
+          io:: Error as NatsError,
```

Comment:
Consider changing `NatsError` ro IOError` since an IO error could come from alot of places other than `nats`

Reply:
Updaetd